arangod/RocksDBEngine/ReplicatedRocksDBTransactionState.cpp

```diff
@@ -231,6 +234,20 @@ void ReplicatedRocksDBTransactionState::addIntermediateCommits(uint64_t value) {
                                  "invalid call to addIntermediateCommits");
 }
 
+arangodb::Result
+ReplicatedRocksDBTransactionState::triggerIntermediateCommit() {
+  THROW_ARANGO_EXCEPTION_MESSAGE(TRI_ERROR_INTERNAL,
```

Comment:
Why is this a THROW and not a return of Result?

return would be expected, throw not